src/AboutPage/About.tsx

```diff
@@ -2,7 +2,46 @@ import React from 'react'
 
```

Comment:
I can't add a comment on the first line for some reason but you can delete the React import because it's not actually used in this file.